client/emv/apduinfo.h

```diff
@@ -31,4 +32,28 @@ typedef struct {
 const APDUCode *GetAPDUCode(uint8_t sw1, uint8_t sw2);
 const char *GetAPDUCodeDescription(uint8_t sw1, uint8_t sw2);
 
+typedef struct {
+    uint8_t cla;
+    uint8_t ins;
+    uint8_t p1;
+    uint8_t p2;
+    uint8_t lc[3];
+} __attribute__((packed)) ExtAPDUHeader;
```

Comment:
Don't we have a define PACKED for this?

Reply:
yes, but i see it in
`include\hitag.h` and `include\pm3_cmd.h`

have added it to `util.h`